### ISSUE TITLE
There seems to be no effect in the road data

### ISSUE THREAD

**Issue by LinkerRrrr** [Kunbyte-AI/ROSE#16]
@m1a0cX Hi,Thank you for your outstanding work.
 I tried to use road data to remove vehicles on the road surface. I obtained vehicle masks through SAM3, but it seems that the vehicles cannot be removed. I want to know why this is the case?
This is the result
https://github.com/user-attachments/assets/4dfc604b-19bd-4399-bceb-6156fbb972d1

**Comment by m1a0cX** [comment]
I think there may be some failure cases. But I found the model totally failed to remove the car which is also out of my expectation. Could you provide the original video and the mask video so I can have a try?

**Comment by LinkerRrrr** [comment]
OK,here they are.Thanks for your reply!

https://github.com/user-attachments/assets/320e1e6a-b186-4af7-9c8b-dae8f9d0cf4e
https://github.com/user-attachments/assets/aa967de5-2812-4bbb-a342-d35dd98de82a

**Comment by m1a0cX** [comment]
> OK,here they are.Thanks for your reply!
> 
> https://github.com/user-attachments/assets/320e1e6a-b186-4af7-9c8b-dae8f9d0cf4e https://github.com/user-attachments/assets/aa967de5-2812-4bbb-a342-d35dd98de82a

I see. I think our current model can't deal with too much masked objects in the video at the same time since our training process only contains single masked object. So when coming to your case the model fails to remove vehicles on the road. To improve the ability of removing multiple objects, I think it needs more training data which have this kind of training sample.

**Comment by LinkerRrrr** [comment]
Alright, then I also want to ask the same question: will the training code be made available?hahaha
